src/cmd/builtin/filemap.c

```diff
@@ -175,8 +175,8 @@ static int subcmd_map (optparse_t *p, int ac, char *av[])
     }
     ctx.p = p;
     ctx.verbose = optparse_get_int (p, "verbose", 0);
-    ctx.chunksize = optparse_get_int (p, "chunksize", default_chunksize);
-    ctx.threshold = optparse_get_int (p,
+    ctx.chunksize = optparse_get_size (p, "chunksize", default_chunksize);
+    ctx.threshold = optparse_get_size (p,
```

Comment:
chunksize and threshold are ints in the context, should be updated?  Not sure of fallout elsewhere in code though.

Reply:
Those values are ints all the way through, so I'd say the option parsing code should make sure the returned value is not greater than INT_MAX as opposed to letting it overflow.  Maybe here a helper like this would be useful
```c
int get_size_int (optparse_t *p, const char *name, const char *default_value)
{
    uint64_t val = optparse_get_size (p, name, default_value);
    if (val == (uint64_t)-1)
        return -1;
    if (val > INT_MAX) {
        optparse_fatal_usage (p, name, 1, "%s must be less than %zu\n", INT_MAX);
        return -1;
    }
    return (int)val;
}

Reply:
and also redo the documentation, again... since obviously `T`, `P`, `E` suffixes are right out.